app/javascript/components/TagList.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <ul v-if="isTagsValid" class="tagList" :class="listClasses">
```

Comment:
Vue has this neat convenience where a `class` property passed into a component will be automatically appended to the class list of its root element. As such, `listClasses` is not necessary:
`<TagList class="some-class" ...>` will automatically apply `some-class` to the component's root element (`ul` in this case).